src/components/home/modals/Send.vue

```diff
@@ -626,12 +637,67 @@ export default {
             }, respErr => {
             })
         },
+        recordTmpRecipient(recipient) {
+            let alias = JSON.parse(window.localStorage.getItem(this.defaultAddress)).alias
+            if (recipient.length < 9 && alias) {
+                for (let key in alias) {
+                    if (recipient.toLowerCase() === alias[key].toLowerCase()) {
+                        this.tmpRecipient = key
+                        break
+                    }
+                }
+            } else {
+                this.tmpRecipient = recipient
+            }
+        },
         inputAmount(num) {
             return BigNumber(num)
         },
+        showRecipient() {
+            if (JSON.parse(window.localStorage.getItem(this.defaultAddress)) != null) {
+                let alias = JSON.parse(window.localStorage.getItem(this.defaultAddress)).alias
+                if (this.recipient.length < 9 && alias) {
+                    for (let key in alias) {
+                        if (this.recipient.toLowerCase() === alias[key].toLowerCase()) {
+                            return this.recipient + ' (' + key + ')'
```

Comment:
`return alias[key] + ' (' + key + ')'` here. Display the alias with upper and lowercase.